libs/ui-lib/lib/common/components/newFeatureSupportLevels/types.ts

```diff
@@ -24,6 +24,7 @@ export type NewFeatureSupportLevelData = {
     featureId: FeatureId,
     supportLevelData?: NewFeatureSupportLevelMap,
     cpuArchitecture?: string,
+    platformType?: string,
```

Comment:
Why is this `string`?
I see this repeating itself over and over again across different files. Can we constraint the type of string we expect  to be assigned to this variable (or parameter, it depends where you look at it)?

Reply:
True, I'll change this to PlatformType.

Reply:
Same with `cpuArchitecture`, while we're at it.

Reply:
Changes done!